php-checks/src/main/java/org/sonar/php/checks/FunctionNameCheck.java

```diff
@@ -77,18 +80,26 @@ private void check(NameIdentifierTree name) {
   private Pattern computePattern() {
```

Comment:
It's actually not computing anything, so renaming it can be good.
```suggestion
  private Pattern getPattern() {
```